### ISSUE TITLE
Entire file is indented, but RuboCop doesn't complain

### ISSUE THREAD

**Issue by bquorning** [bbatsov/rubocop#2001]
```rb
  # encoding: utf-8
  class A
  end
```
RuboCop doesn't discover that all lines should be unindented by 2 spaces.

**Comment by bbatsov** [comment]
That's a nice catch. @jonas054, please have a look at this.

**Comment by jonas054** [comment]
OK!
